feat: separateMultipleMinor (#24538)
renovatebot/renovate: docs/usage/configuration-options.md

e.g. if you are on webpack@v1 currently then default behavior is a PR for upgrading to webpack@v3 and not for webpack@v2.
If this setting is true then you would get one PR for webpack@v2 and one for webpack@v3.

## separateMultipleMinor

> Configure this to `true` if you wish to get one PR for every separate minor version upgrade of a dependency.

Reviewer: the description from the config option will be added here too, so it seems this sentence is duplicate?

> Configure this to `true` if you wish to get one PR for every separate minor version upgrade of a dependency.
> e.g. if you are on webpack@v3.0.0 currently then default behavior is a PR for upgrading to webpack@v3.4.0 and not for webpack@v3.3.0, webpack@v3.2.0 and webpack@v3.1.0
> If this setting is true then you would get one PR for webpack@v3.4.0, webpack@v3.3.0, webpack@v3.2.0 and webpack@v3.1.0

Reviewer 2: ```suggestion
Enable this for dependencies when it's important to split updates into separate PRs per minor release stream (e.g. `python`).

e.g. if you are on python@v3.9.0 currently then default behavior is a PR for upgrading to python@v3.11.0 and not for python@v3.10.0.
If this setting is true then you would get separate PRs for python@3.9.x, python@v3.10.x,  python@v3.11.x
```

e.g. if you are on webpack@v1 currently then default behavior is a PR for upgrading to webpack@v3 and not for webpack@v2.
If this setting is true then you would get one PR for webpack@v2 and one for webpack@v3.

## separateMultipleMinor

Enable this for dependencies when it's important to split updates into separate PRs per minor release stream (e.g. `python`).

> e.g. if you are on python@v3.9.0 currently then default behavior is a PR for upgrading to python@v3.11.0 and not for python@v3.10.0.
> If this setting is true then you would get separate PRs for python@3.9.x, python@v3.10.x,  python@v3.11.x

Reviewer 2: ```suggestion
e.g. if you are on python@v3.9.0 currently then default behavior is a PR for upgrading to `python@v3.11.x` and not for `python@v3.10.x`.
If this setting is true then you would get separate PRs for `python@3.9.x`, `python@v3.10.x`,  `python@v3.11.x`.
```

e.g. if you are on webpack@v1 currently then default behavior is a PR for upgrading to webpack@v3 and not for webpack@v2.
If this setting is true then you would get one PR for webpack@v2 and one for webpack@v3.

## separateMultipleMinor

> Enable this for dependencies when it's important to split updates into separate PRs per minor release stream (e.g. `python`).

Reviewer 3: ```suggestion
Enable this for dependencies when it is important to split updates into separate PRs per minor release stream (e.g. `python`).
```

> e.g. if you are on python@v3.9.0 currently then default behavior is a PR for upgrading to `python@v3.11.x` and not for `python@v3.10.x`.

Reviewer 2: ```suggestion
For example, if you are on python@v3.9.0 currently then default behavior is a PR for upgrading to the latest version such as `python@v3.12.x` and not for `python@v3.10.x`.
```

> If this setting is true then you would get separate PRs for `python@3.9.x`, `python@v3.10.x`, `python@v3.11.x`.

Reviewer 2: ```suggestion
If this setting is true then you would get separate PRs for each minor stream, such as `python@3.9.x`, `python@v3.10.x`, `python@v3.11.x`, etc..
```

Reviewer 3: Nitpick: I don't believe you need to add a period after abbreviations with periods, so just etc. 
```suggestion
If this setting is true then you would get separate PRs for each minor stream, such as `python@3.9.x`, `python@v3.10.x`, `python@v3.11.x`, etc.
```

e.g. if you are on webpack@v1 currently then default behavior is a PR for upgrading to webpack@v3 and not for webpack@v2.
If this setting is true then you would get one PR for webpack@v2 and one for webpack@v3.

## separateMultipleMinor

Enable this for dependencies when it is important to split updates into separate PRs per minor release stream (e.g. `python`).

> For example, if you are on python@v3.9.0 currently then default behavior is a PR for upgrading to the latest version such as `python@v3.12.x` and not for `python@v3.10.x`.
> If this setting is true then you would get separate PRs for each minor stream, such as `python@3.9.x`, `python@v3.10.x`, `python@v3.11.x`, etc

Reviewer 4: ```suggestion
For example, if you are on `python@v3.9.0` currently, then by default Renovate creates a PR to upgrade you to the latest version such as `python@v3.12.x`.
By default, Renovate skips versions in between, like `python@v3.10.x`.

But if you set `separateMultipleMinor=true` then you get separate PRs for each minor stream, like `python@3.9.x`, `python@v3.10.x` and `python@v3.11.x`, etc.
```
